code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -44,7 +60,10 @@
 	if (A)
 		A.air_vent_names -= id_tag
 		A.air_vent_info -= id_tag
-
+	for(var/M in motors)
+		LAZYREMOVE(motors, M)
+		transfer_modifier *= 0.001
```

Comment:
Is there a reason you need to change transfer_modifier here? isn't the pump about to be destroyed?